src/components/ContentHighlights/data/constants.js

```diff
@@ -28,7 +40,19 @@ export const FOOTER_TEXT_BY_CONTENT_TYPE = {
   learnerpathway: 'Pathway',
 };
 
-// Test Data for Content Highlights
+// High Card logic for footer text
+export const getContentHighlightCardFooter = (price, contentType) => {
+  const formattedContentType = FOOTER_TEXT_BY_CONTENT_TYPE[contentType?.toLowerCase()];
+  if (!price) {
+    return formattedContentType;
+  }
+  return `$${price} · ${formattedContentType}`;
+};
+
+// Test Data for Content Highlights TODO: Remove when API is available || Remove when feature completed
+// Test entepriseId for Content Highlights to display card selections and confirmation
+export const testEnterpriseId = 'e783bb19-277f-479e-9c41-8b0ed31b4060';
+// Test Content Highlights data
 export const TEST_COURSE_HIGHLIGHTS_DATA = [
```

Comment:
[curious] Is this constant still in use anywhere at this point? If not, should it be removed?

Reply:
I would advocate to keep it in the the repo for testing purposes. The enterpriseId allows the contents of the Highlights DataTable to be populated allowing for quick testing beyond the local environment with courses and existing meta data about courses.

Feel free to disagree, but I'd strongly encourage leaving it in.